api/location/locationRouter.js

```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const DB = require('../utils/db-helper');
+// const DB = require('../utils/db-helper');
```

Comment:
Same as my comment in the householdRouter.js file, why comment it out when it is no longer needed and can be removed?